CHANGELOG.md

```diff
@@ -1,9 +1,21 @@
+# v1.3.7:
+
+## Features
+- [Feature #216](https://github.com/google/osv-scanner/pull/432)
+  Add SARIF output.
+- [Feature #81](https://github.com/google/osv-scanner/pull/183)
+  Experimental offline mode.
+
+### Minor Updates
+- [Feature #57](https://github.com/google/osv-scanner/pull/432)
+  `--experimental-diff` flag for difference between previous and current run
+  Output is sorted by ID
```

Comment:
```suggestion
```

This has been removed now, it's now a separate not published binary, so no need to put it in the changelog